packages/viewer/src/lib/scenes/camera.ts

```diff
@@ -148,9 +148,8 @@ export abstract class Camera {
     );
     const cameraToCenter = Vector3.subtract(position, boundingBoxCenter);
```

Comment:
If this line were:
```
const cameraToCenter = Vector3.subtract(boundingBoxCenter, position);
```

... then I think you could get rid of the negation below.